jekyll/_cci2/ios-migrating-from-1-2.md

```diff
@@ -0,0 +1,406 @@
+    steps:
+
+      # Get the code from the VCS provider.
+      - checkout
+
+      # Restore all cached files.
+      - restore_cache:
+          keys:
+          - v1-gems-{{ checksum "Gemfile.lock" }}
+          # Fall back to using the latest cache if no exact match is found.
+          - v1-gems-
+
+      # Install dependencies.
+      - run:
+          name: Bundle install
+          command: bundle check || bundle install
+          environment:
+            BUNDLE_JOBS: 4
+            BUNDLE_RETRY: 3
+            BUNDLE_PATH: vendor/bundle
+
+      - save_cache:
+          key: v1-gems-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+
+      - run:
+          name: Install CocoaPods
+          command: bundle exec pod install
+
+      # Run tests.
+      - run:
+          name: Run tests
+          command: bundle exec fastlane scan
+          environment:
+            SCAN_DEVICE: iPhone 6
+            SCAN_SCHEME: WebTests
+
+      # Collect XML test results data to show in the UI,
+      # and save the same XML files under test-results folder
+      # in the Artifacts tab.
+      - store_test_results:
+          path: test_output/report.junit
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: scan-test-results
+      - store_artifacts:
+          path: ~/Library/Logs/scan
+          destination: scan-logs
+
+  deploy:
+    macos:
+      xcode:
+        version: 8.3.3
+
+    steps:
+      - checkout
+
+      # Set up code signing via Fastlane Match.
+      - run:
+          name: Set up code signing
+          command: fastlane match development --readonly
+
+      # Build the release version of the app.
+      - run:
+          name: Build IPA
+          command: bundle exec fastlane gym
+
+      # Store the IPA file in the build artifacts
+      - store_artifacts:
+          path: ./MyApp.ipa
+          destination: ipa
+
+      # Deploy!
+      - run:
+          name: Deploy to App Store
+          command: bundle exec fastlane spaceship
+
+workflows:
+  version: 2
+  build-and-deploy:
+    jobs:
+      - build-and-test
+      - deploy:
+          requires:
+            - build-and-test
+          filters:
+            branches:
+              only: master
+```
+
+## Before you begin
+
+To ensure a consistent build experience, we recommend adding a Gemfile
+and setting up code signing with the help of Fastlane Match before you
+push a 2.0 config to your CircleCI iOS project.
+
+### Gemfile
+
+We suggest that you add a Gemfile to your iOS project if you don't have
+one already. This will ensure that recent versions of Fastlane and
+CocoaPods are available in your build.
+
+A sample Gemfile could look like this:
+
+```
+# Gemfile
+
+source 'https://rubygems.org'
+gem 'fastlane'
+gem 'cocoapods'
+```
+
+### Credentials for code signing via Fastlane Match
+
+For 2.0 we are deprecating our automated code signing support and
```

Comment:
`deprecating` is not the correct word here, since code-signing is currently totally absent. @michelle-luna any ideas on how to phrase this better?